### ISSUE TITLE
Isn it should be I0.rows at errorFunction?

### ISSUE THREAD

**Issue by bona00ku** [facebook/Surround360#255]
At surround360_render/source/optical_flow/PixFlow.h,
I found a mysterious code.

At line 520, when calculating horizontal RegularizationCoef related error, 
shouldn't it be 
                         `horizontalRegularizationCoef * fabsf(flowDir.x) / float(I0.rows);` 
instead of 
                        ` horizontalRegularizationCoef * fabsf(flowDir.x) / float(I0.cols);` 
?

**Comment by aparrapo** [comment]
Nice catch @bona00ku! Please feel free to create a pull request with the fix so we can give you proper recognition :)

**Comment by aparrapo** [comment]
@rumblecoder you are totally right, I read it wrong the first time. Please go ahead and create a pull request. Thanks!

**Comment by rumblecoder** [comment]
Although it makes sense to use I0.rows for vertical regularization and I0.cols for horizontal regularization, my suggestion would conflict with [this](https://github.com/facebook/Surround360/pull/148) previous rejected pull request.
Using I0.cols on both terms was on purpose (a comment in the code here would be helpful). So just reverting the last commit should be fine.
